modules/cudaoptflow/test/test_optflow.cpp

```diff
@@ -468,7 +468,7 @@ CUDA_TEST_P(OpticalFlowDual_TVL1, Async)
 }
 
 INSTANTIATE_TEST_CASE_P(CUDA_OptFlow, OpticalFlowDual_TVL1, testing::Combine(
-    ALL_DEVICES,
+    FIRST_DEVICE,
```

Comment:
I believe there should be a `ts` option to "reconfigure" CUDA `ALL_DEVICES` instead of changing of each tests.

/cc @asmorkalov

Reply:
I would imagine that we could fix the funcion to use the actual device not just gpu0 for all tests.